src/dstack/_internal/core/backends/gcp/compute.py

```diff
@@ -69,11 +73,12 @@ def get_offers(
             availability = InstanceAvailability.NO_QUOTA
             if _has_gpu_quota(quotas[region], offer.instance.resources):
                 availability = InstanceAvailability.UNKNOWN
-            # todo quotas: cpu, memory, global gpu
+            # todo quotas: cpu, memory, global gpu, tpu
             offers_with_availability.append(
                 InstanceOfferWithAvailability(**offer.dict(), availability=availability)
             )
-            offers_with_availability[-1].region = region
+            if not _contains_tpu(requirements):
+                offers_with_availability[-1].region = region
```

Comment:
Why do we show availability zone as the TPU region instead of region?

Reply:
Ok. Now I got the concept of zone and region. Will fix it accordingly.